src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -923,15 +923,27 @@ spring.datasource.username=notification
 spring.datasource.password=notification
 ----
 
-===== Prepare the database purge operation
+===== Database purge operation
 Additionally, a `vacuum` operation does not by default remove orphaned Large Objects from the database.
 Identification and removal of these objects is performed by the `vacuumlo` command.
 
 As the ProActive database contains many large objects, it is essential to apply both `vacuumlo` and `vacuum` operations in order to control the disk space usage over time.
 
-Prior to execute the above operations, a preliminary configuration step is necessary (this preliminary step is done once for a given ProActive installation).
+- **Start the ProActive Server to fill up the database**
 
-After the first start of the ProActive server, the following SQL queries will alter several tables of the database (this is needed due to a PotsgreSQL driver issue):
+Initially, the ProActive Server has be started **at least once** to fill up the ProActive database.
+
+- **Stop the ProActive Server**
+
+Then, once the ProActive Server is up and the web portals become accessible, you have to stop the ProActive Server and proceed to the next operations.
+
+Please, do not stop the Postgres Server.
+
+- **Prepare the database purge operation**
+
+Prior to execute the purge operation, a preliminary configuration step is necessary (this preliminary step is done once for a given ProActive installation).
```

Comment:
This should be moved up, right after the sentence "As the ProActive database contains many large objects, ..."